### PR TITLE
feat(cp-connector): Introduce log forwarding to `cp-connector` library

### PR DESCRIPTION
**Pull request by odubajDT**
## This PR

- introduces log forwarding to cp-connector library

### CODE REVIEW

File: cp-connector/pkg/controlplane/logforwarder_test.go [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+	"testing"
+
+	"github.com/keptn/go-utils/pkg/api/models"
+	"github.com/keptn/go-utils/pkg/common/strutils"
+	keptnv2 "github.com/keptn/go-utils/pkg/lib/v0_2_0"
+	"github.com/keptn/keptn/cp-connector/pkg/controlplane/fake"
+	"github.com/stretchr/testify/require"
+)
+
+func TestLogForwarderNoForward(t *testing.T) {
```
Comment (bacherfl) [inline cp-connector/pkg/controlplane/logforwarder_test.go:13]: let's also add some checks whether `logHandler.Log()` was called or not to each of these test cases?

File: cp-connector/pkg/controlplane/controlplane_test.go [hunk 2]
```diff
@@ -49,8 +69,13 @@ func TestControlPlaneSubscriptionSourceFailsToStart(t *testing.T) {
...
 		return nil
 	}}
+	um := &fake.UniformInterfaceMock{
+		RegisterIntegrationFn: func(integration models.Integration) (string, error) {
+			return "some-id", nil
+		},
+	}
 	integration := ExampleIntegration{RegistrationDataFn: func() RegistrationData { return RegistrationData{} }}
-	err := New(ssm, esm).Register(context.TODO(), integration)
+	err := New(ssm, esm, um).Register(context.TODO(), integration)
```
Comment (bacherfl) [inline cp-connector/pkg/controlplane/controlplane_test.go:78]: here we could also check if the integration id was passed to the eventSource and subscriptionSource

File: cp-connector/pkg/controlplane/logforwarder_test.go (hunk 1 above)
Comment (odubajDT) [reply cp-connector/pkg/controlplane/logforwarder_test.go:13]: adapted, thank you

File: cp-connector/pkg/controlplane/controlplane_test.go (hunk 2 above)
Comment (odubajDT) [reply cp-connector/pkg/controlplane/controlplane_test.go:78]: done

**Review by bacherfl — APPROVED** [review]

File: cp-connector/pkg/controlplane/fake/uniforminterface_mock.go [hunk 3]
```diff
@@ -0,0 +1,33 @@
+package fake
+
+import "github.com/keptn/go-utils/pkg/api/models"
+
+type UniformInterfaceMock struct {
```
Comment (warber) [inline cp-connector/pkg/controlplane/fake/uniforminterface_mock.go:5]: pls rename this file to `uniformapi.go`

**Review by warber — APPROVED** [review]

**Pull request merged**